src/main/java/hudson/plugins/gradle/BuildScanLogScanner.java

```diff
@@ -1,31 +1,37 @@
 package hudson.plugins.gradle;
 
+import java.util.Optional;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 public class BuildScanLogScanner {
+    private static final Pattern BUILD_SCAN_PATTERN = Pattern.compile("Publishing (build scan|build information)\\.\\.\\.");
+    private static final Pattern URL_PATTERN = Pattern.compile(".*(?:\\[INFO] )?(https?://.*/s/.*)");
+    private static final int LINES_TO_SCAN = 1000;
```

Comment:
I think we would need to make it configurable in the future.

Reply:
@guylabs WDYT, do we want to make it configurable and should we wait with this then?

Reply:
I'd keep that for now. If we face another issue in the future we can then always make it configurable. Not worth the efforts for now.

Reply:
Yep, no need to do it now. Something for the future.